corehq/motech/repeaters/models.py

```diff
@@ -448,14 +480,24 @@ def retire(self):
             self['base_doc'] += DELETED_SUFFIX
         self.paused = False
         self.save()
+        # Deleting RepeaterStub will cascade-delete SQLRepeatRecords
+        # and SQLRepeatRecordAttempts too. (RequestLog will still keep a
+        # record of all send attempts.)
+        self.repeater_stub.delete()
+        # NOTE: Undeleting a Repeater needs to include creating a
+        #       RepeaterStub for it.
```

Comment:
In this, pause, and resume, are we concerned about what happens if couch or SQL are not both simultaneously available? Again, using SQL transactions might be useful to mitigate that here.  It might also be useful to have at least one test that illustrates how we handle when one of the systems is not available